src/controller/modules_fn.py

```diff
@@ -106,11 +106,19 @@ def addImages(widget):
     """
     compute addition of all input images
     """
-    parent_names = getParentNames(widget)
-    for parent_name in parent_names:
-        if parent_name not in _IMAGES_STACK.keys():
-            return print("'{}' not in image stack".format(parent_name))
-    im = core.addImages([_IMAGES_STACK[parent_name] for parent_name in parent_names])
+    if widget.singleValue.isChecked():
+        try:
+            value = float(widget.value.text())
+        except ValueError:
+            return print("addition value ERROR")
+        ref_parent_name = widget.reference.currentText()
+        im = core.substractImages(_IMAGES_STACK[ref_parent_name], value=value)
```

Comment:
I thinks that's a mistake, we are in addition no?

Reply:
My mistake